docs/resources/k8s_pool.md

```diff
@@ -104,3 +104,39 @@ Kubernetes pools can be imported using the `{region}/{id}`, e.g.
 ```bash
 $ terraform import scaleway_k8s_pool.mypool fr-par/11111111-1111-1111-1111-111111111111
 ```
+
+## Upgrading a pool
```

Comment:
changing the instance type is different from upgrading a pool; an upgrade on pool means updating system/kubernetes version.